renovate.json

```diff
@@ -0,0 +1,21 @@
+{
+  "commitMessageAction": "update",
+  "commitMessagePrefix": "⬆️(dependencies)",
+  "commitMessageTopic": "{{depName}}",
+  "enabledManagers": ["npm"],
+  "extends": ["config:base"],
+  "packageRules": [
+    {
+      "packagePatterns": [ "*" ],
```

Comment:
Consider grouping major-only dependencies. Otherwise you'll often end up with a build break due to one or more major updates and the whole PR becomes blocked as a result.

Reply:
Weekly is possible too. Also, Renovate now supports Python too now, in case you want to enable it and combine them all into a single PR. Or one for JS and one for Python but reusing common config such as commit prefixes